src/main/twirl/gitbucket/core/account/ssh.scala.html

```diff
@@ -1,9 +1,9 @@
 @(account: gitbucket.core.model.Account, sshKeys: List[gitbucket.core.model.SshKey])(implicit context: gitbucket.core.controller.Context)
 @import gitbucket.core.ssh.SshUtil
-@gitbucket.core.html.main("SSH Keys"){
+@gitbucket.core.html.main("Public SSH Keys"){
```

Comment:
I would keep the left menu as short as possible ; thus not changing it.

Reply:
@McFoggy This is used only for the title of HTML, so this change would be acceptable.